frontend/packages/dev-console/src/components/pipelines/PipelineVisualizationStepList.scss

```diff
@@ -0,0 +1,20 @@
+.odc-pipeline-vis-steps-list {
+  list-style: none;
+  padding: 0;
+  width: 10em;
+  & > li {
```

Comment:
Better to use BEM class names

Reply:
Is it really necessary when using descendent selectors?
I can make the change for consistency.